libr/arch/p/arm/armass64.c

```diff
@@ -84,9 +88,12 @@ typedef struct Opcode_t {
 	Operand operands[MAX_OPERANDS];
 } ArmOp;
 
-#define check_cond(cond) if (!(cond)) { return data; }
+#define check_cond(cond) \
+	if (!(cond)) { \
+		return data; \
+	}
 
-static int get_mem_option(char *token) {
+static int get_mem_option (char *token) {
```

Comment:
function definitions don't have a space before the ( revert all those changes

Reply:
Do you folks have a formatter that you run over the code, I believe these are from clang-d or something.

Reply:
I ran the ./sys/clang-format-diff.py script over the code, hopefully this resolves these kind of issues.